app/assets/stylesheets/components/_firm.scss

```diff
@@ -13,8 +13,10 @@
 }
 
 .firm__adviser-distance {
-  margin-bottom: $baseline-unit*2;
+  margin-left: -$baseline-unit/2;
```

Comment:
I say this also because really we should be avoiding using less than 1 whole unit, otherwise it kind of defeats the purpose